### ISSUE TITLE
iOS Could not create TensorFlow Graph: No OpKernel was registered to support Op 'Switch' with these attrs.

### ISSUE THREAD

**Issue by zen-apps** [tensorflow/tensorboard#795]
I'm working on the sample code for [TensorFlow Lite for iOS](https://github.com/tensorflow/tensorflow/tree/r1.4/tensorflow/examples/ios). I'm able to run the sample just fine. However, when I attempt to load my own model in and labels, I'm getting the following error in XCode:

`Could not create TensorFlow Graph: Invalid argument: No OpKernel was registered to support Op 'Switch' with these attrs.  Registered devices: [CPU], Registered kernels:
 device='GPU'; T in [DT_FLOAT]
 device='GPU'; T in [DT_INT32]
 device='GPU'; T in [DT_BOOL]
 device='GPU'; T in [DT_STRING]
 device='CPU'; T in [DT_INT32]
 device='CPU'; T in [DT_FLOAT]

 [[Node: dropout_3/cond/Switch = Switch[T=DT_BOOL](dropout_1/keras_learning_phase, dropout_1/keras_learning_phase)]]`

I was able to find some updated info [here](https://stackoverflow.com/questions/40855271/no-opkernel-was-registered-to-support-op-switch-with-these-attrs-on-ios), and I attempted this and it didn't work. Any insight?

**Comment by nfelt** [comment]
Closing as this issue doesn't sound like it's specific to TensorBoard (as opposed to TensorFlow).  In general StackOverflow with the tensorboard tag is probably still the best place for this kind of question, but you could also try re-filing this issue in the TensorFlow github issue tracker.

Also, the SO question seems to refer to https://github.com/tensorflow/tensorflow/issues/5919 - maybe that's relevant to your issue?
